src/components/vehicles/forms/components/OwnershipSection.tsx

```
@@ -1,344 +1,183 @@
-  const ownershipStatus = form.watch('ownership_status');
-  
-  const handleOwnershipChange = (value: string) => {
-    // Reset discovery fields when changing from discovered to other status
-    if (value !== 'discovered') {
-      form.setValue('discovery_source', '');
-      form.setValue('discovery_url', '');
-      form.setValue('discovery_location', '');
-    }
+interface OwnershipSectionProps {
+  form: {
+    getValues: () => any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)